app/answerOperations.js

```diff
@@ -0,0 +1,58 @@
+var Question = require('../models/question');
+var Answer = require('../models/answer');
+var textUtils = require('../utils/textutils');
+var tokenUtils = require('../utils/tokenutils');
+var Constants = require('../constants');
+
+module.exports = function () {
+    return setupFunctions();
+};
+
+var setupFunctions = function () {
+    var exports = {};
+
+    exports.postAnswer = function (req, res) {
+        var questionId = req.body.questionId;
+        if (textUtils.isEmpty(questionId)) {
+            res.status(400).json({error: Constants.ERROR.MISSING.QUESTION_ID});
+            console.log("no questionId");
+            return;
+        }
+
+        var text = req.body.answerBody;
+        if (textUtils.isEmpty(text)) {
+            res.status(400).json({error: Constants.ERROR.MISSING.ANSWER_BODY});
+            console.log("no text");
+            return;
+        }
+
+        var token = req.body.token;
+        if (textUtils.isEmpty(token)) {
+            res.status(400).json({error: Constants.ERROR.MISSING.TOKEN});
+            return;
+        }
+
+        tokenUtils.getUserFromToken(token, function (err, doc) { //FIXME there's currently a link between question->answer and answer->question, make it one way
```

Comment:
Something to think about for the future that will make it easier to collab: It will be good to have a consistent way of defining tasks. For instance, 
```
//todo-me A task that is assigned to me and webstorm picks this up
```
Its actually quite a neat feature. Maybe worth checking out: https://www.jetbrains.com/webstorm/help/todo-example.html